docs/generating_coreboot_support_logs.md

```diff
@@ -0,0 +1,186 @@
+Generating coreboot support logs and sending them to verify
+===========================================================
+
+Intro
+-----
+* PLATFORM_IP used as example is `192.168.0.100`
+* PLATFORM_SERIAL_DEVICE used as example is `/dev/ttyUSB0`
+
+Requirements
+------------
+* OS with `cbmem` and `dmesg` installed on the target board
+([`cbmem` installation instruction](./cbmem_building.md))
+* Coreboot gerrit account: https://review.coreboot.org/
+* Possibility to login to `root` user on the target board via SSH or enabled
+coreboot boot logs redirection to the serial console
+* Git (type on target device`apt-get install git -y` to install git on it)
+* Internet connection
+
+Common steps for every generating logs process variant
+------------------------------------------------------
+ 
+1. [Build coreboot ROM](./supported_coreboot_build.md). 
+
+It's important to don't delete ROM image, because script used to generate 
+support logs requires that image. Default directory of built ROM image is
+`coreboot/build/`.
+
+2. [Flash your device with flashrom usage](./flashrom_building.md).
+
+If you have `flashrom` installed on your OS you can jump to 
+`Flashing firmware with flashrom usage` section of instruction linked above.
+
+3. Enter to the `coreboot` directory. 
+
+It's important to run script from that directory, because it's root directory 
+for coreboot scripts. If you try to run script from another directory there 
+is a good chance that an error will be displayed and you will be prompted to 
+change the directory to the correct one.
+
+4. Choose one of the methods shown in sections below and generate coreboot 
+support logs. 
+
+`board_status.sh` is responsible for required logs collecting. 
+It's placed in `coreboot/util/board_status` directory. But don't enter there. 
+You have to stay in `coreboot` directory.
+
+> Sending logs to verify is descibred in `Generating logs and sending them to 
```

Comment:
`sending logs to verify` - it's not correct (`for verification`? what is the meaning of this?) Please rephrase.